Tools/scripts/generate_opcode_h.py

```diff
@@ -129,6 +129,13 @@ def main(opcode_py, outfile='Include/opcode.h'):
         fobj.write("};\n")
         fobj.write("#endif\n")
 
+        fobj.write("\n")
+        fobj.write("#define EXTRA_CASES \\\n")
+        for i, flag in enumerate(used):
+            if not flag and i != 255:
```

Comment:
255 is explicitly marked as used a few lines earlier.

```suggestion
            if not flag:
```